### PR TITLE
[FIX] fix check for incoming trust (RT-3074)

### PR DESCRIPTION
**Pull request by darkdarkdragon**

### CODE REVIEW

**Comment by MatthewPhinney** [comment]
LGTM

**Comment by yongsoo** [comment]
Please hold off on this for now... I'm checking something.

File: src/js/tabs/trust.js [hunk 1]
```diff
@@ -632,8 +632,8 @@ TrustTab.prototype.angular = function (module)
         });
       };
 
-      $scope.isIncoming = function () {
-        return $scope.component.limit_peer._value.t !== 0;
+      $scope.isIncoming = function() {
+        return _.has($scope.component, 'no_ripple_peer');
```
Comment (vhpoet) [inline src/js/tabs/trust.js]: how so?

**Comment by vhpoet** [comment]
@darkdarkdragon there's nothing wrong with the incoming trust check. The problem is when you have both incoming and outgoing trusts it shows 2 buttons.

**Comment by darkdarkdragon** [comment]
@vhpoet can you, please, provide me specific configuration where it appears? I wasn't able to reproduce this with mutual trusts - when you have incoming trust, it doesn't allow you to edit trust.
But what I fixed - is the situation, when you have incoming trust with zero limit and with non-zero balance - in that situation it really shows two buttons.

**Comment by vhpoet** [comment]
@darkdarkdragon That might be the case, balance, and not a trust.. but you fixed it accidentally 'no_ripple_peer' flag doesn't have to do anything with it.

**Comment by darkdarkdragon** [comment]
@vhpoet my bad. Just checked [code](https://github.com/ripple/rippled/blob/develop/src/ripple/rpc/handlers/AccountLines.cpp#L61) - flag "no_ripple_peer" is only set when "Rippling" is turned off on incoming trust line. I've used "no_ripple_peer" to check for incoming connection because if it present, it clearly indicates that trust line is inbound. Just checking limit_peer is not enough now, because limit can be set to zero. 

So I changed code, and now checking for "no_ripple_peer", balance (if negative, when connection is incoming) and for limit_peer.

**Comment by darkdarkdragon** [comment]
rt-3074-1
currently rippletrade allow editing of incoming trust line if limit is set to zero.

**Pull request closed without merging**